api/v1/controllers/perspectives.js

```diff
@@ -160,6 +161,11 @@ module.exports = {
    */
   patchPerspective(req, res, next) {
     doPatch(req, res, next, helper);
+
+    // Remove from cache after successfull PATCH
+    if (featureToggles.isFeatureEnabled('enableCachePerspective')) {
+      redisCache.del('/v1/perspectives');
```

Comment:
I guess I don't remember how perspective caching is working... are you deleting the entire perspective cache when a single perspective gets updated?

Reply:
Yes only one request we cache not any other request we cache in perspective. I guess it makes more sense to remove cache rather then to find that perspective in cache and update it.